services/core-api/app/api/mines/reports/models/mine_report.py

```diff
@@ -221,9 +211,9 @@ def collectRecipients(self, is_proponent):
 
         return core_recipients, ms_recipients
 
-    def getReportSpecificEmailsByReportType(self):
-        art = self.mine_report_definition.compliance_articles[0]
-        notificaiton_list = MineReportNotification.find_contact_by_compliance_article(art.section, art.sub_section, art.paragraph, art.sub_paragraph)
+    def getReportSpecificEmailsByReportType(self, compliance_details):
```

Comment:
not a big deal, just a minor style point- but just to keep in mind for the future, snake_case for python rather than pascalCase for function & variable names.